src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/Button.xaml

```diff
@@ -14,7 +14,7 @@
     >
 
     <Thickness x:Key="ButtonPadding">11,5,11,6</Thickness>
-    <Thickness x:Key="ButtonBorderThemeThickness">1</Thickness>
+    <Thickness x:Key="ButtonBorderThemeThickness">0,0,0,1</Thickness>
```

Comment:
@anjali-wpf changing the border thickness of left, right and top makes the button look like it has a depth in bottom border, but it actually removes the border from left, right and top.